src/apis/AsyncStorage.md

```diff
@@ -8,7 +8,7 @@ wip: true
 type error;
 
 [@bs.scope "AsyncStorage"] [@bs.module "react-native"]
-external getItem: string => Js.Promise.t(Js.Null.t(string)) = "";
+external getItem: string => Js.Promise.t(Js.Null.t(string)) = "getItem";
```

Comment:
This file is not fully up to date with the .re

Reply:
Done